src/test/java/ch/uzh/ifi/hase/soprafs24/service/TaskServiceTest.java

```diff
@@ -406,17 +392,15 @@ public void apply_alreadyApplied_throwsConflict() {
 
         Task task = new Task();
         task.setId(1L);
+        task.setCandidates(new ArrayList<>(Collections.singletonList(candidate)));
```

Comment:
This sets a candidate to the task, so that it's already again when the apply method gets called.